classes/local/helpers/meeting.php

```diff
@@ -234,6 +234,15 @@ public static function bigbluebuttonbn_create_meeting_metadata(&$bbbsession) {
         if ((boolean) config::get('meetingevents_enabled')) {
             $metadata['analytics-callback-url'] = $bbbsession['meetingEventsURL'];
         }
+
+        $courseid = $COURSE->id;
+        $opencastid = (string) $DB->get_field('tool_opencast_series', 'series', array('courseid' => $courseid));
+
+        // If block and series id exists, then add needed series id to BBB metadata
+        if (strlen($opencastid)>=1) {
+            $metadata["opencast-dc-isPartOf"] = $opencastid;
```

Comment:
Is camelCase  really needed?. 

How does the integration with opencast works? is that an post_script that runs on BBB?